testhelp/testhelper.go

```diff
@@ -27,7 +27,8 @@ func IdentityRenamer(name string) string {
 	return name
 }
 
-// Recursive copy src directory below dst directory, with optional transformations.
+// CopyDir recursively copies src directory below dst directory, with optional
```

Comment:
There is not much point in using this to just copy a directory so maybe it would be better named `templatedir` or `fleshoutdir`?

Reply:
To give some background:

I seem to remember that I called it (at the time of Cogito creation, this is just moved) `CopyDir` because there is no function to copy a directory in Go stdlib, so if it is invoked as

CopyDir with [IdentityRenamer](https://github.com/Pix4D/cogito/blob/ccc4876f2fa19e80db0e25489c2bf67536418c6c/testhelp/testhelper.go#L26) and with empty templatedata, it will actually behave as a plain CopyDir.

Having said this, I am still open to renaming, or maybe making this parametric function as private and exposing two public functions, with clearer names and more focused argument list ?

Reply:
Discussion proceeds in comment thread below https://github.com/Pix4D/cogito/pull/88#discussion_r931944930